kmod/src/block.c

```diff
@@ -1297,7 +1297,7 @@ int scoutfs_block_setup(struct super_block *sb)
 	init_waitqueue_head(&binf->waitq);
 	KC_INIT_SHRINKER_FUNCS(&binf->shrinker, block_count_objects,
 			       block_scan_objects);
-	KC_REGISTER_SHRINKER(&binf->shrinker);
+	KC_REGISTER_SHRINKER(&binf->shrinker, "scoutfs-block_info:" SCSBF, SCSB_ARGS(sb));
```

Comment:
The inconsistent _info: or _cache: caught my eye.  Neither are really right, so just drop both.  scoutfs-$subsystem is enough to identify each shrinker.

Reply:
yeah, I went for the underlying `struct` name, but I agree it looked weird. They're now all `block`, `lock` etc. except for `weak_item` ... I guess I could rename that one to `wkic` but that feels weird too.

Reply:
Yeah, "weak_item" looks good to me.